packages/shared/package.json

```diff
@@ -2,25 +2,58 @@
   "name": "@clerk/shared",
   "version": "0.24.4",
   "description": "Internal package utils used by the Clerk SDKs",
-  "types": "./dist/types/index.d.ts",
```

Comment:
I removed the `types` as TS 5 and later might trip over this as it needs different type files for CJS/ESM. So the exports map handles this

Reply:
TS 4 users will be fine with this, as long as they use at least `moduleResolution: "Node16"`. The specific change here was that TS 5 wants to have `.d.ts` and `.d.mts` files